website/templates/project/settings.mako

```diff
@@ -132,15 +130,14 @@
                         <form id="selectAddonsForm">
 
                             % for category in addon_categories:
-
                                 <%
-                                    addons = [
-                                        addon
-                                        for addon in addons_available
-                                        if category in addon.categories
-                                    ]
+                                    addons = []
```

Comment:
I removed the wiki code from the mako template and put it in the python nodes view, and left the old python addons code in the mako template.  Let me know if this works for you.